sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,35 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId, baseURL) {
+
+    let token = this._api._token
```

Comment:
```suggestion
    const token = this._api._token
```

1.  Don't use `let` if you don't reassign the variable. You should set up `eslint` in your IDE to be notified of that.
2.  Don't access "private" properties of other classes (as marked by the `_` prefix). In this case, you should pass the token directly to the gateway class.